Gemfile

```diff
@@ -0,0 +1,11 @@
+source 'https://rubygems.org'
+
+gem 'hashie'
+gem 'rest_client'
+
+group :test do
+  gem 'rspec'
+  gem 'webmock'
+  gem 'pry'
+end
```

Comment:
For 3rd party gems, it's usually better to put dependencies in the `.gemspec` via `add_dependency "hashie"` or `add_development_dependency "rspec"`. Otherwise, your gemfile.lock may cause unresolvable conflicts with a client application.